apps/emqx_authn/src/emqx_authn_password_hashing.erl

```diff
@@ -71,10 +71,10 @@ fields(bcrypt_rw) ->
         [
             {salt_rounds,
                 sc(
-                    integer(),
+                    range(5, 10),
```

Comment:
this will cause compatibility issues?
maybe add a converter which does this: `min(max(Input, 5), 10)`.